lib/smart_app_launch/token_introspection_response_group.rb

```diff
@@ -0,0 +1,127 @@
+      )
+
+      input_instructions %(
+        There are two categories of input for this test group: 
+
+        1. The access token response values, which will dictate what the tests will expect to find in the token 
+        introspection response.  If the Token Introspection Request test group was run AND the option to introspect
+        the access token from the Standalone Launch tests was selected, these values will auto-fill; otherwise,
+        the tester will need to run an out-of-band ACCESS TOKEN request and manually input the access token response
+        parameters.   
+        
+        2. The token introspection response bodies. If the Token Introspection Request test group was run, these will
+        auto-fill; otherwise, the tester will need to an run out-of-band INTROSPECTION requests for:
+
+          a. The ACTIVE access token received from the out-of-band access token request, AND
+
+          b. An INACTIVE OR INVALID token 
+
+        The client making both introspection requests must be authorized to access the introspection endpoint if
+        the endpoint is protected.
+      )
+      
+    test do 
+      title 'Token introspection response for an active token contains required fields'
+
+      description %(
+        This test will check whether the metadata in the token introspection response is correct for an active token and
+         that the response data matches the data in the original access token and/or access token response from the 
+         authorization server, including the following:
+      
+        Required:
+        *  `active` claim is set to true 
+        * `scope`, `client_id`, and `exp` claim(s) match between introspection response and access token
+
+        Conditionally Required:
+        * IF launch context parameter(s) included in access token, introspection response includes claim(s) for 
+        launch context parameter(s) 
+          * Parameters checked for are `patient` and `encounter`
+        * IF identity token was included as part of access token response, `iss` and `sub` claims are present in 
+        introspection response
+
+        Optional but Recommended:
+        * IF identity token was included as part of access token response, `fhirUser` claim SHOULD be present in 
+        introspection response
+      )
+
+      # Access token value is repeatedly overwritten during refresh token tests and value 
+      # when provided for introspection is from the last refresh token exchange test from standalone launch
+      # But other outputs (id token, received scopes, etc) do not update and are still the same as
+      # they were in the original access token request 
+      # Is there any contexts in which this could cause problems, i.e., the response body values from 
+      # using a refresh token grant are different than the response body values from the authorization code grant?
+
+      input :standalone_client_id,
+            title: 'Access Token Respone: client_id'
```

Comment:
Client ID doesn't come from the access token response though, technically, does it?

Reply:
No it doesn't.  See my response about `expires_in` for more details, but I updated the description to say "ID of the client that requested the access token being introspected."